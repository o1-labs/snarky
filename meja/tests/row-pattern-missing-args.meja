let f = fun (x : [> `A(int)]) => {
  switch (x) {
    | `A => ()
    | _ => ()
  };
};
