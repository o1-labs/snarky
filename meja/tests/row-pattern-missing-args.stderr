File "tests/row-pattern-missing-args.meja", line 3, characters 6-8:
Error: Incompatible types ([>`A(int)] as 'a) and ([>`A] as 'b):
       Cannot unify ([>`A(int)] as 'a) and ([>`A] as 'b):
       the constructor `A has different arities.
